theano/gpuarray/c_code/dimshuffle.c

```diff
@@ -0,0 +1 @@
+#section support_code
```

Comment:
NumPy has that limit.  libgpuarray doesn't currently, but I'm thinking about adding it in, since it would simplify some things.